.teamcity/_self/projects/DesktopApp.kt

```diff
@@ -0,0 +1,166 @@
+package _self.projects
+
+import _self.bashNodeScript
```

Comment:
This is a copy/paste from the same project in settings.kts, plus the addition of id("WpDesktop") and `id("WpDesktop_DesktopE2ETests")` so we can rename these objects.